AttitudeManager/Inc/AM_DataTypes.hpp

```diff
@@ -5,17 +5,45 @@
 #ifndef ZPSW3_AM_DATATYPES_HPP
 #define ZPSW3_AM_DATATYPES_HPP
 
-typedef struct {
-    int some_data_here;
-} Position_t;
+#include <cassert>
```

Comment:
Something you pointed out was the need to have our own assert implementation... this is something we should do now before we accidently hardfault in the air